src/target/cortexm.c

```diff
@@ -1265,18 +1265,14 @@ bool cortexm_run_stub(target_s *t, uint32_t loadaddr, uint32_t r0, uint32_t r1,
  * The Flash Patch and Breakpoint (FPB) and Data Watch and Trace (DWT)
  * systems are used. */
 
+/* DWT only supports powers of two as size. Convert length in bytes to
```

Comment:
Same comment as in compat.c about multi-line comments. Please fix this comment in the same way